cmd/src/setup.rs

```diff
@@ -95,18 +95,46 @@ pub fn initial_logger(config: &TiKvConfig) {
                     e
                 );
             });
-            let drainer = logger::LogDispatcher::new(drainer, slow_log_drainer);
-            logger::init_log(
-                drainer,
-                config.log_level,
-                true,
-                true,
-                vec![],
-                config.slow_log_threshold.as_millis(),
-            )
-            .unwrap_or_else(|e| {
-                fatal!("failed to initialize log: {}", e);
-            });
+            if config.rocksdb_log_file.is_empty() {
+                logger::init_log(
+                    drainer,
+                    config.log_level,
+                    true,
+                    true,
+                    vec![],
+                    config.slow_log_threshold.as_millis(),
+                )
+                .unwrap_or_else(|e| {
+                    fatal!("failed to initialize log: {}", e);
+                });
+            } else {
+                let rocksdb_log_drainer = logger::file_drainer(
```

Comment:
if `slow_log_file` is not set, it seems rocksdb log drainer won't be set.

Reply:
Done